### PR TITLE
Updated conference partnership topic with booth guidelines

### PR DESCRIPTION
**Pull request by thatdocslady**
Based on feedback and retrospectives from PyCon, expanded the list of tips and guidelines for booth operation at a conference.

### CODE REVIEW

File: conferences/README.md [hunk 1]
```diff
@@ -47,7 +47,42 @@ Attending your first tech conference can be scary, even when you're not an absol
...
-* When you order swag for the workshop, choose a few key items and order double/triple/etc so that you have enough to give away at the conference. For example, at DevConf.cz we ordered twice the amount of pens and buttons as we gave out during the workshop, and they were gone by the end of the first day!
-* Decorations for the booth are up to you! Some options could be printed tablecloths, rollups, LED lightstrips, and so on. Cupcakes are recommended too ;-)
-* Make sure you have at least 1-2 power adapters/extension cords, as conferences do not always provide these! Optional: Arrange for a computer monitor where you can run a slideshow of previous Django Girls workshop photos, demo the tutorial in action, etc.
+* Write down and pay speciall attention to all the peak times at the booth, such as lunch, coffee breaks, breakfast, and make sure that the core organizers are present during those times.
```
Comment (bmispelon) [inline conferences/README.md]: "special"

File: conferences/README.md [hunk 2]
```diff
@@ -47,7 +47,42 @@ Attending your first tech conference can be scary, even when you're not an absol
...
+
+### Promotion
+
+* Try to have someone on social media duty at the booth, and post lots of pictures! If people are tweeting from their personal accounts, try to have them tag the workshop account.
+* Memorize/prepare a 30 sec - 1 min [elevator pitch](http://en.wikipedia.org/wiki/Elevator_pitch) about Django Girls, as you'll have lots of people asking about what we do and most of the booth time will be spent on these types of introductions.
+
+### Warnings
+
+* Do not leave the booth unattended! Depending on the conference, you might be able to leave basic swag overnight, but anything that is worth money like swag for sale, monitors, and tip jar (with money of course) must be either watched by someone at the booth or taken with you when you leave.
+* Don't waste money on foody swag, and don't be tempted to give away shirts or bags. Unlike the big corporations, we are operating independently and on a donation basis, and conferences provide enough food to all attendies that we don't need to supplament it!
```
Comment (bmispelon) [inline conferences/README.md]: "supplement"

**Comment by thatdocslady** [comment]
@bmispelon - fixed :-)

**Comment by bmispelon** [comment]
Had to do a bit of rebasing magic but I finally merged it in 3345a1194002361ae73a328fcdf9b1de5a09fb2b.

Thanks! :rocket:

**Pull request closed without merging**